src/test/java/uk/ac/ebi/eva/test/utils/JobTestUtils.java

```diff
@@ -88,32 +84,11 @@ public static String getTransformedOutputPath(Path input, String compressExtensi
         return Paths.get(outputDir).resolve(input) + ".variants.json" + compressExtension;
     }
 
-    public static void cleanDBs(String... dbs) throws UnknownHostException {
-        // Delete Mongo collection
-        MongoClient mongoClient = new MongoClient("localhost");
-
-        for (String dbName : dbs) {
-            DB db = mongoClient.getDB(dbName);
-            db.dropDatabase();
-        }
-        mongoClient.close();
-    }
-
     public static JobParameters getJobParameters() {
         return new JobParametersBuilder()
                 .addLong("time", System.currentTimeMillis()).toJobParameters();
     }
 
-    public static File makeGzipFile(String content) throws IOException {
```

Comment:
It's a method to delete that doesn't create temporal files, i've maintained it because it's one of the few tests that can't be modified to use temporals. Is marked as deprected to delete after fixing the problems with the test parameters.